frontend/server/src/main/java/com/amazonaws/ml/mms/http/InferenceRequestHandler.java

```diff
@@ -171,6 +173,7 @@ private static RequestInput parseRequest(
             }
         } else {
             byte[] content = NettyUtils.getBytes(req.content());
+            inputData.setHeaders(acceptMap);
```

Comment:
Headers should be set even for multipart case.

Reply:
Yes, this is the discussion I had with you. I thought we concluded that Accept headers in multipart isn't used as much. I can add it, but handling that is pretty weird. Right now the Accept headers are used at a Request level, but if we handle multipart, then Accept headers will become at a input parameter level.